RecoPixelVertexing/PixelTriplets/plugins/gpuPixelDoubletsAlgos.h

```diff
@@ -110,15 +113,16 @@ namespace gpuPixelDoubletsAlgos {
         continue;
 
       if (doClusterCut) {
-        auto mes = hh.clusterSizeY(i);
```

Comment:
this was the BUG: hiding the outer variable used later in the cut....

Reply:
out of curiosity (will check) - did this not generate any warnings ?

Reply:
so
  - GCC would generate a warning with `-Wshadow` or `-Wshadow=local`, but we are not using them in CMSSW.
  - NVCC does not have these kind of warnings anyway